scripts/notarize.js

```diff
@@ -10,12 +10,14 @@ const notarizeApp = async (context) => {
   const shouldNotarize = process.env.NOTARIZE === 'true';
 
   if (!shouldNotarize || !isMacOS) {
+    // biome-ignore lint/suspicious/noConsoleLog: console.log is used for debugging
```

Comment:
Idea: formalize this into a "debugLog" function, silence biome once, gain better clarity on intent.

Reply:
Done 🤝